outputs.tf

```diff
@@ -1,51 +1,77 @@
+// ----------------------------------------------------------------------------
+// Storage (logs, reports, repo)
+// ----------------------------------------------------------------------------
 output "lts_logs_bucket" {
-  value = module.jx.logs-jenkins-x
+  value       = module.cluster.logs_jenkins_x
+  description = "The bucket where logs from builds will be stored"
 }
 
 output "lts_reports_bucket" {
-  value = module.jx.reports-jenkins-x
+  value       = module.cluster.reports_jenkins_x
+  description = "The bucket where test reports will be stored"
 }
 
 output "lts_repository_bucket" {
-  value = module.jx.repository-jenkins-x
+  value       = module.cluster.repository_jenkins_x
+  description = "The bucket that will serve as artifacts repository"
 }
 
+
+// ----------------------------------------------------------------------------
+// Cluster Name
+// ----------------------------------------------------------------------------
 output "cluster_name" {
-  value = var.cluster_name
+  value       = local.cluster_name
+  description = "The name of the created cluster"
 }
 
+// ----------------------------------------------------------------------------
+// Generated IAM Roles
+// ----------------------------------------------------------------------------
 output "cert_manager_iam_role" {
```

Comment:
Do you really need this as outputs on the top level? And if so do we need to dump the full resource or maybe just its name? 

My take on this was to just print some general output which I thought would be interesting when running `terraform apply`, eg the cluster name and the URLs to the generated buckets.

Reply:
The thing is these outputs are there for the tests, otherwise I'd not consider them.